pkg/rancher-desktop/components/UpdateStatus.vue

```diff
@@ -36,6 +36,26 @@
         <span v-else></span>
       </template>
     </card>
+    <card v-else-if="unsupportedUpgradeAvailable" ref="osUpdateRequired" :show-highlight-border="false">
```

Comment:
The `ref=` doesn't seem to be correct anymore. (I guess that's only for tests?)

Reply:
As far as I can tell, yeah, it's not used. Let me know if you want me to remove it.